tests/builder_test.py

```diff
@@ -4,8 +4,10 @@
 
 import xml.etree.ElementTree as ET
 from xml.dom.minidom import Document
+from freezegun import freeze_time
```

Comment:
https://github.com/xmlrunner/unittest-xml-reporting/blob/6556d194e78e2ddc41ecc2958afcd35317160a04/tests/django_test.py#L11-L26

please use same kind of approach to not run things that depend on freezegun if it is not available